src/api/controllers/fileManager.js

```diff
@@ -34,7 +34,10 @@ router.param('familyId', familyId);
  * /file-manager:
  *   get:
  *     tags: [File-manager]
+ *     summary: Communications list.
+ *     description: Gets communications list. 
```

Comment:
Codacy has a fix for the issue: [Trailing spaces not allowed.](https://app.codacy.com/gh/HEPIA-Association/association-Backend/pullRequest?prid=7909649)
```suggestion
 *     description: Gets communications list.
```